### PR TITLE
Pack Geometries before passing to a web worker

### PR DESCRIPTION
**Pull request by bagnell**
I'm opening this PR early so @mramato can take a look. I'll post performance numbers soon.

### CODE REVIEW

**Comment by bagnell** [comment]
@mramato This is ready for review.

File: Source/Scene/Primitive.js [hunk 1]
```diff
@@ -731,8 +731,14 @@ define([
                     for (i = 0; i < length; ++i) {
                         geometry = instances[i].geometry;
                         instanceIds.push(instances[i].id);
+
+                        var moduleName = geometry.constructor.name;
+                        if (moduleName === "Object") {
```
Comment (mramato) [inline Source/Scene/Primitive.js]: Is this to handle pre-created geometry?

**Comment by pjcozzi** [comment]
Did we have performance stats for this?

**Comment by mramato** [comment]
In Chrome, an average of 5 runs loading `ne_10m_admin_0_countries.json` (drag and drop wall time; me and @bagnell both have this topjson file) takes `13.05 seconds` in master and `10.71 seconds` in this branch, which is about an 18% speed-up.  I would still like to do some more testing with various use cases.

**Comment by mramato** [comment]
I've played with this enough that I'm happy bringing it into master (plus it will have plenty of time to stew before the next release).  Despite the size of the PR, it's actually not that big of a change (lots of boilerplate code for pack/unpack).  I'll update CHANGES and merge.

**Pull request merged**